apps/nuxt3-ssr/utils/harmonization.ts

```diff
@@ -49,14 +83,14 @@ const calcStatusForSingleVariable = (
   }
 };
 
-const calcStatusForRepeatingVariable = (
+const calcStatusForAggregatedRepeatingVariable = (
   variable: IRepeatingVariableWithMapping,
   cohort: { id: string }
 ): HarmonizationStatus => {
   const statusList = !variable.repeats
     ? []
     : variable.repeats.map((repeatedVariable) => {
-        const resourceMapping = variable.mappings?.find((mapping) => {
+        const resourceMapping = repeatedVariable.mappings?.find((mapping) => {
```

Comment:
this change is the main bug fix for the overall harmonisation matrix ( maybe also port to old catalogue)